eng/scripts/InstallationCheck.ps1

```diff
@@ -0,0 +1,32 @@
+param (
+    [Parameter()]
+    [string] $ArtifactsDirectory,
+
+    [Parameter()]
+    [string] $Artifact,
+
+    [Parameter()]
+    [string] $PipelineWorkspace
+)
+
+Write-Host "dotnet new console"
+dotnet new console
+$localFeed = "$PipelineWorkspace/$ArtifactsDirectory-signed/$Artifact"
+Write-Host "dotnet nuget add source $localFeed"
+dotnet nuget add source $localFeed
+
+$version = (Get-ChildItem "$localFeed/*.nupkg" -Exclude "*.symbols.nupkg" -Name).replace(".nupkg","").replace("$Artifact.","")
```

Comment:
While this might work in some cases it isn't the easiest or best way to get the version number. We always publish a packageinfo.json file in the artifacts as well that we generally use to read the metadata and version of the artifacts.